news/_posts/2021-05-31-prestashop-1-7-8-0-beta-release.md

```diff
@@ -0,0 +1,240 @@
+
+- The id_group_shop is shared instead of id_shop_group in webservice parameter [#19566](https://github.com/PrestaShop/PrestaShop/issues/19566)
+
+**CO**
+
+-.docker folder is present in release zip [#22233](https://github.com/PrestaShop/PrestaShop/issues/22233)
+
+### Notable technical improvements
+
+**Hooks**
+- New hooks into Presenters, allowing to enrich the data built by these services [#11125](https://github.com/PrestaShop/PrestaShop/issues/11125)
+- Ability to enable and disable Hooks [#20848](https://github.com/PrestaShop/PrestaShop/issues/20848)
+- Hook for modifying TinyMce [#19408](https://github.com/PrestaShop/PrestaShop/issues/19408)
+
+- Remove microdata in HTML flow and replace by JSON-LD for classic theme [#22867](https://github.com/PrestaShop/PrestaShop/issues/22867)
+- Automatize some of our rules with PHPStan: Introduce phpstan extension [#22728](https://github.com/PrestaShop/PrestaShop/issues/22728)
+- Problem with sessions cookies for SameSite issue [#22711](https://github.com/PrestaShop/PrestaShop/issues/22711)
+- Two service containers coexist in the back-office [#14995](https://github.com/PrestaShop/PrestaShop/issues/14995)
+- Add hooks to cart modal - crosseling and promotion [#17709](https://github.com/PrestaShop/PrestaShop/issues/17709)
+- Use static analyzer to improve code quality [#16471](https://github.com/PrestaShop/PrestaShop/issues/16471)
+- No High DPI support in ImageRetriever [#19798](https://github.com/PrestaShop/PrestaShop/issues/19798)
+- Allow override of checkoutProcess construction #19848 [#19848](https://github.com/PrestaShop/PrestaShop/issues/19848)
+- BO Menu translation [#9762](https://github.com/PrestaShop/PrestaShop/issues/9762) 
+- Added the ability to manage and download Attachments via Webservice [#12728](https://github.com/PrestaShop/PrestaShop/pull/12728)
+- CSS classes / element attributes called by core.js are marked with js prefixed selectors [#14346](https://github.com/PrestaShop/PrestaShop/issues/14346)
+- Import source files of the UIKit instead of dist files to use prestakit variables [#18396](https://github.com/PrestaShop/PrestaShop/pull/18396)
+- Default theme is the only folder not having ESLint installed and configured [#22998](https://github.com/PrestaShop/PrestaShop/issues/22998)
+- Default theme has no specific SCSS rules [#22885](https://github.com/PrestaShop/PrestaShop/issues/22885)
+- Add selectors mapping so themes can override it [#20002](https://github.com/PrestaShop/PrestaShop/pull/20002)
+- Add eslint to themes and classic folders [#20080](https://github.com/PrestaShop/PrestaShop/pull/20080)
+- Update BO theme style in legacy pages to match the UI kit [#22435](https://github.com/PrestaShop/PrestaShop/issues/22435)
+- Update jQuery on classic, FO core and default BO theme [#23122](https://github.com/PrestaShop/PrestaShop/pull/23122)
+- Introduce Javascript component loading through window.prestashop (part1) [#20591](https://github.com/PrestaShop/PrestaShop/pull/20591)
```

Comment:
```suggestion
**Front-Office**
- Hook for modifying TinyMce [#19408](https://github.com/PrestaShop/PrestaShop/issues/19408)
- Add hooks to cart modal - crosseling and promotion [#17709](https://github.com/PrestaShop/PrestaShop/issues/17709)
- CSS classes / element attributes called by core.js are marked with js prefixed selectors [#14346](https://github.com/PrestaShop/PrestaShop/issues/14346)
- Import source files of the UIKit instead of dist files to use prestakit variables [#18396](https://github.com/PrestaShop/PrestaShop/pull/18396)
- Default theme is the only folder not having ESLint installed and configured [#22998](https://github.com/PrestaShop/PrestaShop/issues/22998)
- Default theme has no specific SCSS rules [#22885](https://github.com/PrestaShop/PrestaShop/issues/22885)
- Add selectors mapping so themes can override it [#20002](https://github.com/PrestaShop/PrestaShop/pull/20002)
- Add eslint to themes and classic folders [#20080](https://github.com/PrestaShop/PrestaShop/pull/20080)
- Update BO theme style in legacy pages to match the UI kit [#22435](https://github.com/PrestaShop/PrestaShop/issues/22435)
- Update jQuery on classic, FO core and default BO theme [#23122](https://github.com/PrestaShop/PrestaShop/pull/23122)
- Introduce Javascript component loading through window.prestashop (part1) [#20591](https://github.com/PrestaShop/PrestaShop/pull/20591)

**Back-Office**
- New hooks into Presenters, allowing to enrich the data built by these services [#11125](https://github.com/PrestaShop/PrestaShop/issues/11125)
- Ability to enable and disable Hooks [#20848](https://github.com/PrestaShop/PrestaShop/issues/20848)
- Problem with sessions cookies for SameSite issue [#22711](https://github.com/PrestaShop/PrestaShop/issues/22711)
- Two service containers coexist in the back-office [#14995](https://github.com/PrestaShop/PrestaShop/issues/14995)
- No High DPI support in ImageRetriever [#19798](https://github.com/PrestaShop/PrestaShop/issues/19798)
- Allow override of checkoutProcess construction #19848 [#19848](https://github.com/PrestaShop/PrestaShop/issues/19848)
- BO Menu translation [#9762](https://github.com/PrestaShop/PrestaShop/issues/9762) 

**CO**
- Remove microdata in HTML flow and replace by JSON-LD for classic theme [#22867](https://github.com/PrestaShop/PrestaShop/issues/22867)
- Automatize some of our rules with PHPStan: Introduce phpstan extension [#22728](https://github.com/PrestaShop/PrestaShop/issues/22728)
- Use static analyzer to improve code quality [#16471](https://github.com/PrestaShop/PrestaShop/issues/16471)

**WebServices**
- Added the ability to manage and download Attachments via Webservice [#12728](https://github.com/PrestaShop/PrestaShop/pull/12728)
```